client-app/shared/checkout/components/select-address-modal.vue

```diff
@@ -65,11 +65,72 @@
       </div>
     </template>
 
+    <div v-if="showFilters" class="select-address-modal__filters">
```

Comment:
The component should follow a consistent approach. Please choose:

1. Add complete BEM structure in this PR
2. Keep Tailwind-only and migrate to BEM in a separate task later (please create such a task)

Partial BEM implementation is not acceptable.